_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,38 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it. These outputs may be useful in particular circumstances, e.g. L2s, when the parent transaction cannot have fees for technical or accounting reasons.
 
+- Recommend reading from earlier review clubs:
+  - [#30352: Add PayToAnchor(P2A), OP 1 <0x4e73> as standard output script for spending](https://bitcoincore.reviews/30352)
 
 ## Questions
 
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
 
+1. Where in code is it determined if an amount is dust?
+
+1. Is dust restricted by consensus? Policy? Both?
+
+1. How can dust be problematic?
+
+1. What is _ephemeral_ dust?
+
+1. What are the specific proposed rules for ephemeral dust?
+
+1. Why is it important to impose a fee restriction?
+
+1. How are 1P1C relay and TRUC transactions relevant to ephemeral dust?
+
+1. What could happen if restrictions on the the child tx were not in place?
+
+1. Which types of tests were created for the PR? What is the purpose of each type of test?
+
+1. Why is the test in [this commit](https://github.com/bitcoin/bitcoin/pull/30239/commits/fd22f9e2f4709a00bd31b739a7edadf98a728831) important?
```

Comment:
I think the answer to this question is pretty much copy-pasting the commit message, so perhaps it'd be good to approach this from a different angle? Perhaps something along the lines of the following could be a more adventurous alternative: "what happens when we broadcast a transaction with properties <...> and a node with `-dustrelayfee=0`? How do we protect against this / what makes you confident this won't be an issue \  ..."

Reply:
Yeah, instagibbs did a great job of thoroughly explaining things in commit logs and comments. I rephrased the line of questions and removed direct reference to the commit. The original goal of this question was for new reviewers to understand the importance of avoiding regression by creating thorough tests that defensively cover feature interactions, which I can talk to during the meeting.